docs/tokens.md

```diff
@@ -30,11 +36,19 @@ TxData {
 }
 ```
 
-## Issue Tokens
-When issuing a new token, we specify the data for creating a new token in the transaction input, where the `token_id` is a hash of the inputs. **TODO which inputs?**
-**TODO explain remaining fields**
+## Issuing Tokens
+
+**TODO is the sentence below true? In the example, I don't see any data relating to the token in the input.** 
+
+When issuing a new token, we specify the data for creating a new token in the transaction input, where the `token_id` is a hash of the inputs.
```

Comment:
When issuing a new token, we specify the data for creating a new token in the transaction **output**. After that, a wallet should calculate the `token_id` that is a hash of the first input of the issuance transaction. And use this token_id is able to send tokens to anyone.

Reply:
Thanks, that clarifies things! I will modify the explanation.